### ISSUE TITLE
valueBox class not always being cleared

### ISSUE THREAD

**Issue by davidski** [rstudio/flexdashboard#106]
When setting the `color` attribute based upon the value of the input to `valueBox` when using the Shiny runtime, the valuebox div does not always have the previous class removed and winds up with two `bg-*` classes applied (and the wrong value taking precedence).

To reproduce:
1) Run the following example (or check out the live running version on [Shinnyapps](https://davidski.shinyapps.io/flextest/))
2) Choose `danger` and see the color change as expected
3) Choose `success` and see the color change as expected
4) Choose `danger` again and see the color **not** change. Inspecting the HTML shows two classes now applied.

Example code as follows:
```
---
title: "Shiny testbed"
output: 
  flexdashboard::flex_dashboard:
    orientation: columns
    vertical_layout: fill
runtime: shiny
---

```{r setup, include=FALSE}
library(flexdashboard)
library(tidyverse)
```

Column {data-width=650}
-----------------------------------------------------------------------

### Chart A

```{r}
data(mtcars)
ggplot(mtcars, aes(mpg, cyl)) + geom_point()
```

Column {data-width=350}
-----------------------------------------------------------------------

### Input Selection

```{r}
class_color <- c("warning", "danger", "success")
selectInput("input_value", "Choose Color", class_color)
```

### Datatable

```{r}
options(DT.fillContainer = FALSE)
options(DT.autoHideNavigation = FALSE)
DT::renderDataTable({
 dat <- mtcars
 DT::datatable(dat, options = list(
    scrollY = "200px"
  ))
})
```

### Valuebox
```{r}
renderValueBox({
  dat <- input$input_value
  valueBox(dat,
           caption = "Testing colors", icon = "fa-pencil", 
           color = dat)
})
```
```

**Comment by jjallaire** [comment]
I've seen this bug before but had hard time reproing it -- this drove me crazy! Thanks so much, I'll see if I can get it fixed based on your example.

**Comment by Felixmil** [comment]
@jjallaire  This bug seems to be back.

I reused the the example presented here and got the behavior described by @davidski above.

I thinks it's because the value box class are now `value-box-danger` and not `bg-danger` .

I tried to fork the repo and modify the lines that changed in 5bd738c but couldn't  find the file `inst/rmarkdown/templates/flex_dashboard/resources/flexdashboard.js`. How does it work now ?
